xlsx/src/import/worksheets/mod.rs

```diff
@@ -0,0 +1,535 @@
+use core::str;
+use std::{collections::HashMap, io::Read, num::ParseIntError};
```

Comment:
I put all the common parsing code between both implementations here. I think everything in this file just came directly from the old tree-based parser, but we need a common place so the new streaming parser can call into these functions.